react-ui/src/project/index.js

```diff
@@ -199,7 +199,8 @@ const Project = () => {
                         </div>
                     ) : null}
                     {selectedValue === "Burndown Chart" && isBurndown ? (
-                        <SprintDetail sprintDetails={data.sprints} attributes={data.custom_attributes} token={auth} projectName={data.name} />
+                        // <SprintDetail sprintDetails={data.sprints} attributes={data.custom_attributes} token={auth} projectName={data.name} />
+                        <Graph type="Burndown Chart" />
```

Comment:
Remove this line. We will handle it during integration. Otherwise it will cause failure.

Reply:
Did